dashboard/src/components/Header.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div>
+    <b-navbar toggleable type="dark" variant="dark">
+      <span>
+        <b-navbar-toggle target="nav-collapse" class="header-brand"></b-navbar-toggle>
+        <b-navbar-brand to="/">MHP DAShboard</b-navbar-brand>
+      </span>
+    </b-navbar>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'header',
+};
+</script>
+
+<style>
```

Comment:
All components that are not `App.vue` should have `<style scoped>`. See https://vuejs.org/v2/style-guide/#Component-style-scoping-essential
Should prevent us with having global styles applied everywhere (unless neccessary)